src/core_plugins/elasticsearch/lib/clusters.js

```diff
@@ -0,0 +1,14 @@
+import Cluster from './cluster';
+
+const clusters = new Map();
+
+export function getCluster(name) {
+  return clusters.get(name);
+}
+
+export function createCluster(name, config) {
+  const cluster = new Cluster(config);
+  clusters.set(name, cluster);
```

Comment:
This should probably validate that it's not overriding existing clusters of the same name.